internal/mode/static/state/change_processor_test.go

```diff
@@ -1590,15 +1557,6 @@ var _ = Describe("ChangeProcessor", func() {
 				changed, _ := processor.Process()
 				Expect(changed).To(BeTrue())
 			})
-			It("should report not changed after multiple Upserts of the resource with same generation", func() {
```

Comment:
`change_processor.go` should not handle any cases with the same generation, as these should already be filtered out by the controller's `GenerationChangedPredicate`.